src/Engine/MapController.java

```diff
@@ -641,6 +647,16 @@ public XYCoord getContemplationCoord()
     return myGameInputHandler.getCoordinateOptions();
   }
 
+  public enum OverlayMode
+  {
+    THREATS, GUNS, VISION, NONE,
```

Comment:
Took me a while to figure out that `THREATS` means "only threats which are specifically flagged by the user". Based on the name alone, it seems potentially duplicative of `GUNS`.

Reply:
Yeah. I thought about putting "user" or "enemy" something in some of them, but I couldn't figure a way to word it.
Any ideas?

Reply:
I dunno, maybe `TAGGED` or `MANUAL` vs `ALL`.